### PR TITLE
Update wingpanel

### PR DESCRIPTION
**Pull request by lucasnota**

### CODE REVIEW

**Comment by alucryd** [comment]
wingpanel-0.4.x-bzr:
- No need to create yet another wingpanel package, the 0.4.x branch still points to rewrite-x11 and the current package is still valid.
- Wrong description.
- The pantheon-shell group does not exist, I picked pantheon and pantheon-unstable a while ago, to reflect gnome in our repos.
- Why add gtk-update-icon-chache? There is no icon in that package...

wingpanel-indicator-ayatana-bzr:
- Depends line is wrong.
- Empty prepare function.
- Again, why gtk-update-icon-cache...
- Anyway, I'm not packaging an ayatana compat layer chen the goal is to get rid of ayatana. Ayatana indicators are never gonna end up in our repos anyway.

wingpanel-indicator-datetime-bzr:
- Install file? Really?

wingpanel-indicator-network-bzr:
- Hmm, why change network to power in MKPKG?
- Yes libnm-gtk is the same as nm-connection-editor and I should probably use the latter. I'm kinda aiming for alphabetical order though. Also, whitespaces.

wingpanel-indicator-notifications-bzr:
- You're breaking my pkgdesc scheme.
- wingpanel already deps on granite, no need for redundancy
- Why provide and conflict with the same pkgname...
- Why rename the source with the exact same name?

wingpanel-indicator-session-bzr:
- Again, alphabetical order.
- Why remove gobject-introspection? Did you even try to build it without it? And by without, I mean clean chroot or uninstalled from your system. Copying the vapi files doesn't fix anything.
- Install file...

wingpanel-indicator-sound-bzr:
- Install file......

tl;dr:
This PR is half-baked and essentially moot, apart from issues it doesn't add much. I appreciate the thought but I'll be closing it.
